src/model/rpc.js

```diff
@@ -1,141 +1,162 @@
 const EventEmitter = require('events')
-const uuid = require('uuid')
-const request = require('superagent')
-const Clp = require('clp-packet')
+const clpPacket = require('clp-packet')
+const ilpPacket = require('ilp-packet')
```

Comment:
Shouldn't these two start with an uppercase? See https://gist.github.com/hueniverse/a06f6315ea736ed1b46d#module-classes and http://nodeguide.com/style.html#class-names

Reply:
No, according to [StandardJS](https://standardjs.com/rules.html), constructor functions have to start with a capital letter. `clp-packet` and `ilp-packet` are no constructor functions. [javascript.crockford.com/code.html](http://javascript.crockford.com/code.html) has a neat explanation why this rule exists: 

>  Constructor functions that must be used with the new prefix should start with a capital letter. JavaScript issues neither a compile-time warning nor a run-time warning if a required new is omitted. Bad things can happen if new is not used, so the capitalization convention is the only defense we have.